### PR TITLE
Deploy check50 v2.0.0

### PR DESCRIPTION
**Pull request by brianyu28**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "Programming Language :: Python :: 3",
+        "Topic :: Education",
+        "Topic :: Utilities"
+    ],
+    description="This is check50, with which you can check solutions to \
+        problems for CS50.",
+    install_requires=["argparse", "pexpect", "requests", "backports.shutil_which", "termcolor"],
+    keywords=["check", "check50"],
+    name="check50",
+    py_modules=["check50", "config"],
```
Comment (dmalan) [inline setup.py]: Do we (still) need `config`?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    author_email="sysadmins@cs50.harvard.edu",
+    classifiers=[
+        "Intended Audience :: Education",
+        "Programming Language :: Python :: 3",
+        "Topic :: Education",
+        "Topic :: Utilities"
+    ],
+    description="This is check50, with which you can check solutions to \
+        problems for CS50.",
+    install_requires=["argparse", "pexpect", "requests", "backports.shutil_which", "termcolor"],
```
Comment (dmalan) [inline setup.py]: Requires `submit50` no?

Comment (brianyu28) [inline setup.py]: Added!

File: setup.py (hunk 1 above)
Comment (brianyu28) [inline setup.py]: Yeah, per https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules, it's still the canonical way to share variables between different modules.

Comment (dmalan) [inline setup.py]: k

**Pull request merged**